src/fitnesse/responders/run/SuiteResponder.java

```diff
@@ -292,10 +292,32 @@ protected File getRerunPageFile() {
   }
 
   protected String getRerunPageName() {
-    PageCrawler pageCrawler = page.getPageCrawler();
-    WikiPagePath fullPath = pageCrawler.getFullPath();
-    String fullPathName = PathParser.render(fullPath);
-    return "RerunLastFailures_"+fullPathName.replace(".","-");
+	    PageCrawler pageCrawler = page.getPageCrawler();
```

Comment:
@Eqerem-Hena you changed the indentation of these lines (and now it no longer matches that of other methods). Can you please fix this?